code/game/vehicles/spacepods/spacepod.dm

```diff
@@ -250,9 +250,12 @@
 
 /obj/spacepod/attackby(obj/item/W as obj, mob/user as mob, params)
 	if(iscrowbar(W))
-		hatch_open = !hatch_open
-		playsound(loc, 'sound/items/Crowbar.ogg', 50, 1)
-		to_chat(user, "<span class='notice'>You [hatch_open ? "open" : "close"] the maintenance hatch.</span>")
+		if(!equipment_system.lock_system || allow2enter || hatch_open)
```

Comment:
the only problem is that the lock system could be removed or allow2enter could be modified by the pilot. So you may have to jump in, unlock the doors, then get out to open the pod without the lock system check. The hatch open thing is just a QoL thing for operators who forget to close said hatch before locking their pod. Any suggestions for making this cleaner would be appreciated though.